code/datums/datumvars.dm

```diff
@@ -998,6 +998,33 @@
 			manipulate_organs(C)
 			href_list["datumrefresh"] = href_list["editorgans"]
 
+		else if(href_list["givemartialart"])
+			if(!check_rights(NONE))
+				return
+
+			var/mob/living/carbon/C = locate(href_list["givemartialart"]) in GLOB.mob_list

```

Comment:
GLOB.carbon_list iirc

Reply:
I thought this would mess up the carbon check, but it works fine. Good find.